qa-jcoin.planx-pla.net/etlMapping.yaml

```diff
@@ -15,26 +15,67 @@ mappings:
         props:
```

Comment:
the portal config will need to be updated; tests results below

```
[2022-04-29 18:50:36,615][ gen3utils][   INFO] Validating portal config file qa-jcoin.planx-pla.net/portal/gitops.json
[2022-04-29 18:50:37,521][ gen3utils][  ERROR]   - Field error: Field [drug_used] in explorerConfig.filters.tabs.fields not found in etlMapping
[2022-04-29 18:50:37,522][ gen3utils][  ERROR]   - Field error: Field [ever_transferred] in explorerConfig.filters.tabs.fields not found in etlMapping
[2022-04-29 18:50:37,522][ gen3utils][  ERROR]   - Field error: Field [pregnancy_status] in explorerConfig.filters.tabs.fields not found in etlMapping
[2022-04-29 18:50:37,522][ gen3utils][  ERROR]   - Field error: Field [health_insurance] in explorerConfig.filters.tabs.fields not found in etlMapping
[2022-04-29 18:50:37,522][ gen3utils][  ERROR]   - Field error: Field [center] in explorerConfig.filters.tabs.fields not found in etlMapping
[2022-04-29 18:50:37,522][ gen3utils][  ERROR]   - Field error: Field [project_id] in explorerConfig.filters.tabs.fields not found in etlMapping
[2022-04-29 18:50:37,522][ gen3utils][  ERROR]   - Field error: Field [project_id] in explorerConfig.table.fields not found in etlMapping
[2022-04-29 18:50:37,522][ gen3utils][  ERROR]   - Field error: Field [project_id] in explorerConfig.table.fields not found in etlMapping
```

Reply:
@paulineribeyre  yup - this is the next thing on the list - although would like to get the DD and ETL into PROD first if that's ok?

Reply:
@paulineribeyre  will delete these now before merge

Reply:
the explorer page will be broken in prod. Are you sure that's ok?

Reply:
it sounds like you want to deploy to prod today - since the DD changes require a migration of the data, has the old data been deleted in prod already? are you going to re-submit new data to prod today?

i just want to make sure that broken data and a broken page in prod are ok with you, since i can't guarantee someone from my team will be available to help before your visit on Monday if something is broken

Reply:
@paulineribeyre  Ah right -- ok so I made above portal config changes in QA so this should be ready to go -- have not deleted the old data yet in PROD so will do this now before deploying to PROD